sound/soc/sof/ipc4-topology.c

```diff
@@ -985,6 +985,48 @@ static int sof_ipc4_widget_assign_instance_id(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+/* update hw_params based on the audio stream format */
+static int sof_ipc4_update_hw_params(struct snd_sof_dev *sdev, struct snd_pcm_hw_params *params,
+				     struct sof_ipc4_audio_format *fmt)
+{
+	struct snd_interval *i;
+	struct snd_mask *m;
+	snd_pcm_format_t snd_fmt;
+	unsigned int channels, rate;
+	int valid_bits;
```

Comment:
I would have used different order of the definitions to comply with our coding style.